app/filters.tsx

```diff
@@ -69,6 +69,14 @@ export const stateFilters: StateFilter[] = [
       answerFinalizedTimestamp_lt: nowTimestamp,
     },
   },
+  {
+    name: 'Finalizing',
```

Comment:
I'm not sure what the correct name for it is; it would just be nice if we could get to these markets somehow.